### ISSUE TITLE
Adding types for organizations balance sheet informations

### ISSUE THREAD

**Issue by philLaine** [schemaorg/schemaorg#1783]
As far as I know, at the moment Schema.org don't have appropriate type to describe organizations **balance sheet informations**.
I meen financial results, accounting balance sheet, normalized EBITDA, and those sorts of data.

I'm working for a major French company who's providing free financial online data about all the french organizations, I woul like to improve a better schema.org tagging process.
I think this could help finding and sharing the data.

I didn't achieve to join https://www.w3.org/community/schemaorg/ so I'm trying to launch the idea here.

Those new types could be a part of the Organization type.

What do you think ? What could be the next step ?

**Comment by thadguidry** [comment]
@philLaine We don't have OrganizationalFinance types or CompanyStock types yet (these have been discussed a bit in the past on our mailing list but not much)... we only have FIBO types basically to cover common Banking actions. http://schema.org/docs/financial.html  As far as I know, there hasn't been anyone really taking the lead to align schema with OrganizationalFinance or CompanyStock ontologies or vocabularies.  But keep this issue open so others can begin getting energized to do something about that.

@philLaine Are you familiar with http://www.ifrs.org/issued-standards/list-of-standards/ ??

**Comment by philLaine** [comment]
@thadguidry thanks a lot for your quick and clear answer about my request.
I've taked a look, before posting that, at the recent FIBO types. They are great but not accurate to fit my requirements.
I think I could try to take the lead, with my team of developers, to align Schema with OrganizationalFinance ontologies.
As a part of Dun and Bradstreet we have the required legitimacy and the team to support and lead the movement.
I'll ask some help around me and let you know.

Thanks about the IFRS information. I'm not familiar with it. The issue is that balance sheets are different between France and USA for example. It's a nice challenge to fine a vocabularie who could fit to any countries !

Is there someone here who could bring is expertise of Schema.org new types to this project ?

**Comment by philLaine** [comment]
@thadguidry thanks a lot for your quick and clear answer about my request.
I've taked a look, before posting that, at the recent FIBO types. They are great but not accurate to fit my requirements.
I think I could try to take the lead, with my team of developers, to align Schema with OrganizationalFinance ontologies.
As a part of Dun and Bradstreet we have the required legitimacy and the team to support and lead the movement.
I'll ask some help around me and let you know.

Thanks about the IFRS information. I'm not familiar with it. The issue is that balance sheets are different between France and USA for example. It's a nice challenge to fine a vocabularie who could fit to any countries !

Is there someone here who could bring is expertise of Schema.org new types to this project ?

**Comment by philLaine** [comment]
@thadguidry thanks for your quick answer.

Infortunately, FIBO types aren't accurate with my aim. 
So we'll ha ve to create a new vocabulary to describe balance sheet informations types.

Thanks for the IFRS information.
I'll have to ask my expert colleagues from Dun & BradStreet and Les Echos to join the group, because we'll have to find an international vocabulary, who could be able to work with every country accounting specificities...

Is there someone here who could bring us is experience with Schema/org new types creation process ?

**Comment by RichardWallis** [comment]
@philLaine I may be able to help with guidance in this area.  As a matter of interest I was part of the team that created and proposed the FIBO Schema.org terms.

Contact me off line if you feel I could be of help:  rjw @ dataliberate.com

~Richard.

**Comment by thadguidry** [comment]
@philLaine @RichardWallis Yeah, if you are not familiar with IFRS then you DEFINITELY want to start your research there between the 2 of you.  TONS of work as gone into that standard and collaboration between countries and its been I think like a 10 year effort since the first ideas before an official project and name was wrapped around it.  IFRS might not be totally agreeable with you on some of the assumptions they made, but that comes with any democratic compromise that has to happen globally.....the same compromises that we in Schema.org sometimes have to take. 

OH, and if either of you need some deep clarity you can submit an official inquiry through here: http://www.ifrs.org/supporting-implementation/how-the-ifrs-interpretations-committee-helps-implementation/

Best of luck guys.  I'll be in the background.

**Comment by philLaine** [comment]
Thanks a lot @RichardWallis and @thadguidry 
So the project is on the way !
I'll let you know when I'll have some news and a better vision of each steps.
